cmd/gandalf/main.go

```diff
@@ -46,7 +46,7 @@ const (
 	DefaultRepo   = "vic"
 	DefaultBranch = "master"
 
-	DefaultFellows = "caglar10ur,mhagen,mwilliamson,fabio"
+	DefaultFellows = "mhagen,mwilliamson,ghicken"
```

Comment:
@mdharamadas1 We'll need to change this in the binary that runs in CI.

Reply:
Yeah, thanks for the heads-up! I will update it after this is merged.